pyaedt/modeler/cad/Primitives.py

```diff
@@ -2737,28 +2739,34 @@ def _check_material(self, matname, defaultmatname):
             Name of the material.
         defaultmatname : str
             Name of the default material to assign if ``metname`` does not exist.
+        threshold : float
+            Threshold conductivity to distinguish dielectric from conductor.
+            Default value is 100000 S/m.
 
         Returns
         -------
-        str or bool
-            String if a material name, Boolean if the material is a dielectric.
+        (str, bool)
+            Material name, Boolean True if the material is a dielectric, otherwise False.
```

Comment:
```suggestion
            Material name and ``True`` if the material is a dielectric or ``False`` otherwise.
```
I assume it returns both--but as separate return values, not a list.